lib/pages/home_body_content.dart

```diff
@@ -46,32 +45,35 @@ class HomeBodyContent extends StatelessWidget {
   }
 
   List<Widget> buildList(BuildContext context, AsyncSnapshot snapshot) {
```

Comment:
can you abstract this into a new widget file?

Reply:
moved